### PR TITLE
Adding PowerBIDedicated RP swagger

### PR DESCRIPTION
**Pull request by tarostok**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by tarostok** [comment]
@veronicagg following our mail thread appreciate if you could approve this pull request.
Thanks,
Tal

File: specification/powerbidedicated/resource-manager/readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+
+### Tag: package-2017-10-01
+
+These settings apply only when `--tag=package-2017-10-01` is specified on the command line.
+
+``` yaml $(tag) == package-2017-10-01'
+input-file:
+- Microsoft.PowerBIDedicated/package-2017-10-01/powerbidedicated.json
```
Comment (veronicagg) [inline specification/powerbidedicated/resource-manager/readme.md]: realizing that this path looks incorrect, as I noticed a comment in the CI job about not finding the file in the config file. (https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285536553#L617)
"package" is not part of the folder name where the api version is, path should be "Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json", could you address this?

**Review by veronicagg — CHANGES_REQUESTED** [review]
This a move of the PR from the private repo, reviewed and approved by API board, ARM and SDK teams. 
Left just one comment of something I missed before, rest looks good.

**Comment by veronicagg** [comment]
Referencing PR from private repo corresponding this one: https://github.com/Azure/azure-rest-api-specs-pr/pull/133

File: specification/powerbidedicated/resource-manager/readme.md (hunk 1 above)
Comment (tarostok) [reply specification/powerbidedicated/resource-manager/readme.md]: Thank for catching that!
Fixed.

File: specification/powerbidedicated/resource-manager/readme.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ tag: package-2017-10-01
 
 These settings apply only when `--tag=package-2017-10-01` is specified on the command line.
 
-``` yaml $(tag) == package-2017-10-01'
+``` yaml $(tag) == 'package-2017-10-01'
 input-file:
-- Microsoft.PowerBIDedicated/package-2017-10-01/powerbidedicated.json
+- Microsoft.PowerBIDedicated/2017-10-01/powerbidedicated.json
```
Comment (veronicagg) [inline specification/powerbidedicated/resource-manager/readme.md]: sorry about the delayed reply, I think capitalization is not the same as in the folder structure, therefore reference is not getting properly picked up by CI https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285933140#L617
We're still running the validation on the file, but the link to the readme file, seems not to be working. 
Do you mind double-checking on this?

**Review by veronicagg — CHANGES_REQUESTED** [review]

File: specification/powerbidedicated/resource-manager/readme.md (hunk 2 above)
Comment (tarostok) [reply specification/powerbidedicated/resource-manager/readme.md]: update path in readme file, hope it fixes the issue.
Thanks!

**Comment by veronicagg** [comment]
@tarostok great, thanks!

**Review by veronicagg — APPROVED** [review]

**Pull request merged**